File: ZamolxesLegacy 2011/src/BaseAssets/Button.java
package BaseAssets;

public class Button {
	float x,y;
	static int id=0;
	
	public Button(){
		id++;
	}
	
	public void setPivot(float x,float y){
		this.x=x;
		this.y=y;
	}
	
	public float getX() {
		return x;
	}

	public void setX(float x) {
		this.x = x;
	}

	public float getY() {
		return y;
	}

	public void setY(float y) {
		this.y = y;
	}

	public int getId() {
		return id;
	}

	@SuppressWarnings("static-access")
	public void setId(int id) {
		this.id = id;
	}
	
	
}

File: NemesisMapEditor/Button.cpp
#include "Button.h"

Button::Button(int _x,int _y)
{
                   x=_x;
                   y=_y;
}

void Button::SetX(int _x)
{
     x=_x;
}

void Button::SetY(int _y)
{
     y=_y;
}

int Button::GetX()
{
    return x;
}

int Button::GetY()
{
    return y;
}

